submissions/2024-11-19-OpenFF-NAGL2-Training-Optimization-Dataset-Part-1-v4.0/README.md

```diff
@@ -0,0 +1,57 @@
+# OpenFF NAGL2 Training Optimization Dataset Part 1 v4.0
+
+## Description
+A dataset containing molecules from the [`MLPepper RECAP Optimized Fragments v1.0`](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-07-26-MLPepper-RECAP-Optimized-Fragments-v1.0)
+and [`MLPepper RECAP Optimized Fragments v1.0 Add Iodines`](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-11-MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0) datasets,
+with additional conformers and optimized at the OpenFF default level of theory (B3LYP-D3BJ/DZVP).
```

Comment:
This might be a minor point, but would you call these "additional conformers" or did you replace all of the conformers? From my reading of the notebook you only kept the CMILES and then regenerated conformers from scratch.